centaur/src/main/resources/standardTestCases/curl/curl.wdl

```diff
@@ -8,7 +8,7 @@ task url_grab {
         File progressBar = stderr()
     }
     runtime {
-        docker: "curlimages/curl:latest"
+        docker: "google/cloud-sdk"
```

Comment:
pourquoi?

Reply:
The task failed when I was using the curlimages one, and passed when I used `cloud-sdk`.

Reply:
hard to argue with that.